### PR TITLE
Replace queue macro with inline functions

### PR DESCRIPTION
**Pull request by just-now**

### CODE REVIEW

File: test/unit/test_conn.c [hunk 1]
```diff
@@ -25,10 +25,11 @@ TEST_MODULE(conn);
 
 static void connCloseCb(struct conn *conn)
 {
-	bool *closed = conn->queue[0];
+	uint64_t *closed = (uint64_t *)conn->queue.next;
 	*closed = true;
 }
 
+/* Note: "attribute aligned" suppresses clunky cast warning */
```
Comment (cole-miller) [inline test/unit/test_conn.c:32]: There's no `__attribute__((aligned))` in the diff, is this comment stale?

File: test/unit/test_conn.c [hunk 2]
```diff
@@ -37,7 +38,7 @@ static void connCloseCb(struct conn *conn)
 	FIXTURE_RAFT;     \
 	FIXTURE_CLIENT;   \
 	struct conn conn; \
-	bool closed;
+	uint64_t closed;
```
Comment (cole-miller) [inline test/unit/test_conn.c:41]: why not just make this field `queue *`?

**Review by cole-miller — CHANGES_REQUESTED** [review]

**Comment by cole-miller** [comment]
@just-now could you rebase this? (Sorry for the delay)

File: test/unit/test_conn.c (hunk 2 above)
Comment (just-now) [reply test/unit/test_conn.c:41]: Overall solution with bool closed is "hackerish". I'd prefer not to make it even more convoluted.

**Review by cole-miller — APPROVED** [review]

**Pull request merged**